src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocleadingasteriskalign/InputJavadocLeadingAsteriskAlignCorrect.java

```diff
@@ -0,0 +1,97 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocLeadingAsteriskAlign" />
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocleadingasteriskalign;
+
+/** Title
+ * Javadoc for class
+ */
+public class InputJavadocLeadingAsteriskAlignCorrect {
+
+  /** javadoc for instance variable. */
+  private int a;
+
+  /** */
+  private int b;
+
+  /***/
+  private String str;
+
+  /** */
+  private String str1;
+
+  /**
+    This javadoc is allowed because there is no leading asterisk.
+   */
+  public InputJavadocLeadingAsteriskAlignCorrect() {}
+
+  /***
+   * @param a testing....
+   */
+  public InputJavadocLeadingAsteriskAlignCorrect(int a) {}
+
+  /*************************************************
+   *** @param str testing.....
+   **********************************/
```

Comment:
detected these false negatives in the diff report, fixed them and added them to the input test file.